locales/en/translation.json

```diff
@@ -1,34 +1,961 @@
+  "Clear Encrypt Password": "Clear Encrypt Password",
+  "Advanced": "Advanced",
+  "Sync Wallets Across Devices": "Sync Wallets Across Devices",
+  "Create a Shared Wallet": "Create a Shared Wallet",
+  "Use more than one device to create a multisig wallet": "Use more than one device to create a multisig wallet",
+  "Join a Shared Wallet": "Join a Shared Wallet",
+  "Joining another user's multisig wallet requires an invitation to join": "Joining another user's multisig wallet requires an invitation to join",
+  "Buy ": "Buy ",
+  "Don't risk losing your money": "Don't risk losing your money",
+  "Your recovery key is composed of 12 randomly selected words. Take a couple of minutes to carefully write down each word in the order they appear.": "Your recovery key is composed of 12 randomly selected words. Take a couple of minutes to carefully write down each word in the order they appear.",
+  "I'M SURE": "I'M SURE",
+  "Write down each word.": "Write down each word.",
+  "Payment Request": "Payment Request",
+  "Share this QR code to receive in your wallet .": "Share this QR code to receive in your wallet .",
+  "Receive ": "Receive ",
+  "Confirm ": "Confirm {{title}}",
+  "Enable Replace-By-Fee": "Enable Replace-By-Fee",
+  "Edit gas limit": "Edit gas limit",
+  "Nonce": "Nonce",
+  "Destination Tag": "Destination Tag",
+  "Edit destination tag": "Edit destination tag",
+  "A Destination Tag is an optional number that corresponds to an invoice or a XRP account on an exchange.": "A Destination Tag is an optional number that corresponds to an invoice or a XRP account on an exchange.",
+  "Speed Up": "Speed Up",
+  "Slide to send": "Slide to send",
+  "Could not send transaction": "Could not send transaction",
+  "BitPay Card": "BitPay Card",
+  "BitPay Visa&reg; Card ()": "BitPay Visa&reg; Card ()",
+  "Network Cost": "Network Cost",
+  "Miner fee": "Miner fee",
+  "BY USING THIS CARD YOU AGREE WITH THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. This card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Mastercard International. \"Metropolitan Commercial Bank\" and \"Metropolitan\" are registered trademarks of Metropolitan Commercial Bank ©2014.": "BY USING THIS CARD YOU AGREE WITH THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. This card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Mastercard International. \"Metropolitan Commercial Bank\" and \"Metropolitan\" are registered trademarks of Metropolitan Commercial Bank ©2014.",
+  "Mastercard is a registered trademark and the circles design is a trademark of Mastercard International Incorporated.": "Mastercard is a registered trademark and the circles design is a trademark of Mastercard International Incorporated.",
+  " Gift Card": " Gift Card",
```

Comment:
As commented before, we need to add {{context}} to the key in order to avoid this kind of issue. Instead to modify only value, the key should be === value. 
```
 "{{displayName}} Gift Card": "{{displayName}} Gift Card"
```